test/gif.test.js

```diff
@@ -11,7 +12,9 @@ chai.should();
 
 chai.use(chaiHttp);
 
-const token = jwt.sign({ userId: 101, isAdmin: true, email: 'my@gmail.com' }, process.env.JWT_SECRET);
+const token = jwt.sign({ userid: 90, isAdmin: true, email: 'hea@gmail.com' }, process.env.JWT_SECRET);
+const tokenn = jwt.sign({ userid: 100, isAdmin: false, email: 'headies@gmail.com' }, process.env.JWT_SECRET);
```

Comment:
why do we have double n

Reply:
to generate another token to test if a user is not an admin